persistence/persistence-mybatis/src/main/resources/org/cbioportal/persistence/mybatis/SampleListMapper.xml

```diff
@@ -30,6 +30,7 @@
         </include>
         FROM sample_list
         INNER JOIN cancer_study ON sample_list.CANCER_STUDY_ID = cancer_study.CANCER_STUDY_ID
+        INNER JOIN reference_genome ON cancer_study.REFERENCE_GENOME_ID = reference_genome.REFERENCE_GENOME_ID
```

Comment:
because it has .StudyMapper.select., StudyMapper queries the name of reference genome.
```
        <if test="projection == 'DETAILED'">
            ,
            <include refid="org.cbioportal.persistence.mybatis.StudyMapper.select">
                <property name="prefix" value="${prefix}cancerStudy."/>
            </include>
        </if>
```